lib/chromic_pdf/pdf/browser/session_pool.ex

```diff
@@ -188,8 +188,9 @@ defmodule ChromicPDF.Browser.SessionPool do
   # Reasons we want to gracefully clean up the target in the Browser:
   # - max_session_uses_reached, our own mechanism for keeping memory bloat in check
   # - error, when an exception is raised in the Channel
+  # - DOWN, client link is broken (reported by a user as "this happens")
   def terminate_worker(reason, worker_state, pool_state)
-      when reason in [:max_session_uses_reached, :error] do
```

Comment:
I guess we can't easily test this, right?

Reply:
Not really, no. One way would be

1. Spawn a task in the test, run `print_to_pdf` with a long PDF
2. Shoot this task :gun: 
3. Assert that the session is restarted (see `assert_target_respawn` in `SessionPoolTest`)

not sure if I have the motivation to try it, though

Reply:
Was just asking. I think it is all fine without that 🤷

Reply:
tested now